<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GroupBoxRecipeConfig</class>
 <widget class="QWidget" name="GroupBoxRecipeConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>460</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="1">
       <widget class="QComboBox" name="comboBoxSelectRecipe"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelRecipeTargetsMetStatus">
        <property name="text">
         <string>Targets met status</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QPushButton" name="pushButtonPublishRecipe">
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelSelectRecipe">
        <property name="text">
         <string>Change recipe:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="labelRecipeDescription">
        <property name="text">
         <string>Standard recipe</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QTableWidget" name="tableWidgetRecipeComponents"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
